### ISSUE TITLE
World Control No Crafting Recipes in JEI

### ISSUE THREAD

**Issue by Hyperion505** [World-Control-Team/World-Control#18]
Using the 1.12 alpha build while running 1.12.2 Modern Skyblock 3 Departed.

Trying to view the items recipes comes up with nothing interface wise.

Tried googling around to see if this issue was known or not. I do realize its in alpha for this version and it could also be an issue with the modpack, but thought i could throw this out there.

**Comment by dmf444** [comment]
Yep, just me being stupid. Recipes will be added next release

**Comment by Crosus97** [comment]
still no having recipes in JEI

**Comment by xJon** [comment]
Any plans to add the recipes?...

**Comment by xbony2** [comment]
feel free to use minetweaker or another tool

**Comment by xJon** [comment]
@xbony2 I do, but it is nicer to have the mod provide some default recipes that give a general idea of the original design!

**Comment by BDAthans** [comment]
I agree, would be really nice to have the recipes built into the mod for usage in a survival server.
